libs/compute_api/src/requests.rs

```diff
@@ -5,6 +5,11 @@ use crate::privilege::Privilege;
 use crate::responses::ComputeCtlConfig;
 use crate::spec::{ComputeSpec, ExtVersion, PgIdent};
 
+#[derive(Clone, Debug, Deserialize, Serialize)]
```

Comment:
Please add a comment, to explain what this is, which API it's used in

Reply:
Updated to include a mention of the external HTTP server.